projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -115,7 +115,7 @@ function wpcom_launchpad_get_task_definitions() {
 		),
 		'setup_general'                      => array(
 			'get_title'            => function () {
-				return __( 'Set up your site', 'jetpack-mu-wpcom' );
```

Comment:
@arthur791004, can we add some sort of conditional logic to fall back on the existing string until we have translations for the new strings? The new code will likely be shipped to Simple sites later today, and Atomic sites on Tuesday, which may be before we have translations for the new string.

I don't think there are any easy helper functions to achieve this, so we may need to do something like the following:
```suggestion
				$updated_string  = __( 'Give your site a name', 'jetpack-mu-wpcom' );

				if ( $updated_string !== 'Give your site a name' ) || str_starts_with( get_locale(), 'en' ) ) {
					return $updated_string;
				}

				return __( 'Set up your site', 'jetpack-mu-wpcom' );				
```

Reply:
It's not a new string so the translation should be ready I guess.

https://github.com/Automattic/jetpack/blob/b5a21898c4d919b9012eafa900300d260d65b6eb/projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php#L321

Reply:
Awesome! Thanks for finding the existing string and re-using it!